Session/Home/Message Requests/MessageRequestsViewController.swift

```diff
@@ -6,7 +6,11 @@ import SessionMessagingKit
 
 @objc
 class MessageRequestsViewController: BaseVC, UITableViewDelegate, UITableViewDataSource {
-    private var threads: YapDatabaseViewMappings!
+    private var threads: YapDatabaseViewMappings! =  {
+        let result = YapDatabaseViewMappings(groups: [ TSMessageRequestGroup ], view: TSThreadDatabaseViewExtensionName)
+        result.setIsReversed(true, forGroup: TSMessageRequestGroup)
```

Comment:
Good catch - I completely missed that they were oldest first

Reply:
Cheers! I was thinking of applying the same change to home screen, but I'm not sure if this can work for home screen. I think we'll refactor home screen when we refactor the database.

Reply:
Mm yea that'd be good - I expect there's going to be the opportunity to do a lot of refactoring when we swap out the database lol